test/builder/lib/QuarkBuilderTest.sol

```diff
@@ -102,18 +103,28 @@ contract QuarkBuilderTest {
             morphoPositions: emptyMorphoPositions_(),
             morphoVaultPositions: emptyMorphoVaultPositions_()
         });
+        Accounts.QuarkSecret[] memory quarkSecrets = new Accounts.QuarkSecret[](3);
+        quarkSecrets[0] = quarkSecret_(address(0xb0b), BOB_DEFAULT_SECRET);
+        quarkSecrets[1] = quarkSecret_(address(0xa11ce), ALICE_DEFAULT_SECRET);
+        quarkSecrets[2] = quarkSecret_(address(0xfe11a), FELLA_DEFAULT_SECRET);
+
+        address[] memory accounts = new address[](3);
+        accounts[0] = address(0xb0b);
+        accounts[1] = address(0xa11ce);
+        accounts[2] = address(0xfe11a);
+
         chainAccountsList[1] = Accounts.ChainAccounts({
             chainId: 8453,
-            quarkSecrets: quarkSecrets_(address(0xb0b), BOB_DEFAULT_SECRET),
-            assetPositionsList: assetPositionsList_(8453, address(0xb0b), uint256(amount / 2)),
+            quarkSecrets: quarkSecrets,
+            assetPositionsList: assetPositionLists_(8453, accounts, uint256(amount / 2)),
```

Comment:
Shouldn't you make these same changes to `chainAccountsList[0]` as well?

Reply:
oh ok.  I thought it was purposedly make each chains to hold different accounts.  
I think from very early the chainAccountsList[0] (chain 1) is always having different set of accounts in chainAccountList[1] (chain 8453)